COPYING-LGPL

```diff
@@ -0,0 +1,175 @@
+Files in this code repository are to be licensed under the LGPLv3, which you'll
+find below.
+
+However, you'll find some files that carry a license header that assigns them as
+GPLv3, which was the default license for VOLK up to version 3. You can find the
```

Comment:
Could you change this to `before version 3`? That'd be more verbose. At least for me.